examples/java/spring-boot/src/main/java/uk/gov/api/springboot/controllers/MetadataController.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.api.springboot.controllers;
+
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.api.springboot.models.metadata.ApiMetadata;
+import uk.gov.api.springboot.services.MetadataService;
+
+@RestController
+@RequestMapping("/apis")
+public class MetadataController {
+
+  private MetadataService service;
```

Comment:
We should be using `final` for fields:

```suggestion
  private final MetadataService service;
```